### ISSUE TITLE
add a Jitter effect as an option?

### ISSUE THREAD

**Issue by oisintheblue** [valadaptive/ntsc-rs#35]

**Comment by valadaptive** [comment]
Can you describe what this effect would do in more detail?

**Comment by oisintheblue** [comment]
the subtle effect that makes the video go up and down

**Comment by Degamisu** [comment]
> the subtle effect that makes the video go up and down

This is already integrated as the `use field` when its set to `alternating`. this takes the interlacing fields and alternates between them, making a 'jitter' effect

**Comment by Degamisu** [comment]
> > the subtle effect that makes the video go up and down
> 
> This is already integrated as the `use field` when its set to `alternating`. this takes the interlacing fields and alternates between them, making a 'jitter' effect

Now that i think about it, this would be a good idea. Alternating fields would not help much on this one. Appologies!

**Comment by valadaptive** [comment]
@oisintheblue Do you have any example footage of this on a real VHS transfer? In my head, I guess vertical jitter is associated more with silent-era film stock than VHS.

**Comment by Degamisu** [comment]
> @oisintheblue Do you have any example footage of this on a real VHS transfer? In my head, I guess vertical jitter is associated more with silent-era film stock than VHS.

No, VHS is still film. Depending on the tracking, bad tracking is able to make the picture "jump", however jittering can also just happen normally. This happens with all old tapes and films. mostly horizontal jitter though.

**Comment by Degamisu** [comment]
well to rephrase that, there should be a tracking slider. The worse tracking, the worse the picture gets. For example, color desaturation and much vertical jittering.

**Comment by valadaptive** [comment]
I'm tempted to close this unless examples can be provided of VHS footage that actually does this. I've never seen this except in cases where the tracking is *really* messed up and the video is almost unwatchably corrupted.

**Comment by 0x5066** [comment]
I have a hunch that they're talking about this:
https://youtu.be/gClHbBd05QI&t=39
https://www.youtube.com/watch?v=OfsotonUjRM
